packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -3039,6 +3039,31 @@ const ruleset = {
                 function: noDuplicatePathsForScopeParameter,
             },
         },
+        SystemDataDefinitionsCommonTypes: {
+            description: "Must use the schema provided in the common types for SystemData.",
```

Comment:
Is there a reason we are not using the same description as in `rules.md` ? Something like:

> System data references must utilize common types.

In general, I am unsure why we would want for the description in `rules.md` be mismatched with the error description.

Reply:
Changed them to that